src/pm/hydra/tools/topo/hwloc/topo_hwloc.c

```diff
@@ -172,55 +172,188 @@ static struct hwloc_obj_info_table hwloc_obj_info[] = {
     {NULL, HWLOC_OBJ_TYPE_MAX}
 };
 
-static HYD_status handle_bitmap_binding(const char *binding, const char *mapping)
+static HYD_status get_hw_obj_list(const char *mapping, hwloc_obj_t ** map_list,
```

Comment:
Are these argument names accurate anymore? Seems we are trying to get a list of objects from a string of format `object:count`. Not necessarily a mapping/map_list.

Reply:
We get mapping object list as well as the number of objects and the hwloc object type.

Reply:
What I'm trying to say is that we pass both `binding` and `mapping` into the `mapping` argument. It seems like a slight mismatch in naming. Just a nitpick. Does that make sense?

Reply:
ah yes, i'll rename the arguments

Reply:
@raffenet I have updated the variable names.

Reply:
:+1: